index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="author" content="Oleksandr Starov">
+    <meta charset="UTF-8">
+    <title>Amazing Maze</title>
+
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css" integrity="sha384-fLW2N01lMqjakBkx3l/M9EahuwpSfeNvV63J5ezn3uZzapT0u7EYsXMjQV+0En5r" crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/style.css" type="text/css" />
+    <script type="text/javascript" src="./js/utils.js" defer></script>
+    <script type="text/javascript" src="./js/view.js" defer></script>
+    <script type="text/javascript" src="./js/mazeCell.js" defer></script>
+    <script type="text/javascript" src="./js/maze.js" defer></script>
+    <script type="text/javascript" src="./js/main.js" defer></script>
+    
+</head>
+<body>
+    <h1><p>Maze</p></h1>
+    <div id="mainMenu">
+        <p>Please select size for a new maze. After a click you will see a required maze size generation process</p>
+        <button id="small" class="btn btn-success" onclick="startGeneration(this.id)"><span class="glyphicon glyphicon-th-large"></span> Small</button>
+        <button id="medium" class="btn btn-warning" onclick="startGeneration(this.id)"><span class="glyphicon glyphicon-th"></span> Medium</button>
+        <button id="large" class="btn btn-danger" onclick="startGeneration(this.id)"><span class="glyphicon glyphicon-qrcode"></span> Large</button>
+    </div>
```

Comment:
Yep, 100% agree. I just wanted to use this approach to be familiar with this solution.